### PR TITLE
feat: Add Accessibility Statement

### PR DESCRIPTION
**Pull request by rhysbarrett**
### JIRA

[Accessibility statement](https://companieshouse.atlassian.net/browse/SR-106)

### Change Description

Adds the Accessibility Statement page, and links to it from the service footer.

### Footer Screenshot

Screenshot 2020-10-22 at 15 54 57

### Work Checklist

- [x] Tests added where applicable
- [x] Test coverage of new code meets target of 80%
- [x] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### CODE REVIEW

File: src/views/accessibility-statement.njk [hunk 1]
```diff
@@ -0,0 +1,172 @@
+{% extends 'layout.njk' %}
+
+{% block pageTitle %}
+  Accessibility statement for the Apply to remove your home address from the Companies House register service
+{% endblock %}
+
+{% block beforeContent %}{% endblock %}
```
Comment (akaJSaunders) [inline src/views/accessibility-statement.njk:7]: do we need this empty block?

File: test/ApplicationFactory.ts [hunk 2]
```diff
@@ -35,6 +36,8 @@ export function createApp(authEnabled?: boolean) {
   app.set('views', viewPath);
   app.set('view engine', 'njk');
 
+  app.locals.paths = Paths;
```
Comment (akaJSaunders) [inline test/ApplicationFactory.ts:39]: From my understanding, we are only using one link from this locals.paths, do we really need to save all of the Paths?  

Could we save the accessibility path only, and give the variable a more descriptive name? What do you think

File: test/HtmlPatternAssertions.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 export function expectToHaveLink(body: string, href: string, linkText: string): void {
-  const patternStr = `<a.* href="${href}">\\s*${linkText}\\s*<\/a>`;
+  const patternStr = `<a.* href="${href}"\( target="_blank"\|\)>\\s*${linkText}\\s*<\/a>`;
```
Comment (akaJSaunders) [inline test/HtmlPatternAssertions.ts:2]: We want to make sure this still matches links that dont open new tabs, could we double check?

File: src/views/accessibility-statement.njk (hunk 1 above)
Comment (n-r-b) [reply src/views/accessibility-statement.njk:7]: looks like this is hiding the beta banner

File: test/ApplicationFactory.ts (hunk 2 above)
Comment (n-r-b) [reply test/ApplicationFactory.ts:39]: do you mean for the tests only?

Comment (n-r-b) [reply test/ApplicationFactory.ts:39]: the `locals.paths` is used in other places too

File: test/HtmlPatternAssertions.ts (hunk 3 above)
Comment (n-r-b) [reply test/HtmlPatternAssertions.ts:2]: from what i can see, we're only covering links that open in a new tab in our tests currently

**Review by akaJSaunders — APPROVED** [review]

**Pull request merged**